modules/clair-clairctl-standalone.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// clair/master.adoc
+
+:_content-type: PROCEDURE
+[id="clair-disconnected-standalone-configuration"]
+= Setting up a standalone deployment of Clair for a disconnected {ocp} cluster
+
+Use the following procedures to set up a standalone deployment of Clair for a disconnected {ocp} cluster.
+
+[id="clair-clairctl-standalone"]
+== Installing the clairctl command line utility tool for standalone Clair deployment on {ocp}
+
+Use the following procedure to install the `clairctl` CLI tool for standalone Clair deployments on {ocp}.
+
+.Procedure
+
+. Install the `clairctl` program for a standalone Clair deployments by using the `podman cp` command, for example:
```

Comment:
@hdonnay Is there a place we can publish `clairctl` instead of having users copy it out of a container?

Reply:
We publish it to github releases -- I don't think there's a blessed Red Hat way to distribute non-container, non-RPM software.